docs/getting-started.md

```diff
@@ -23,7 +23,7 @@
 
 - [Kubectl](https://kubernetes.io/docs/tasks/tools/) installed.
 - An active Kubernetes cluster, accessible using kubectl.
-- [Antrea](https://github.com/antrea-io/antrea/) deployed. Recommend v1.10.
+- [Antrea](https://github.com/antrea-io/antrea/) deployed. Recommend v1.9.
```

Comment:
We are comptabile with 1.10 now.. Dont change the version

Reply:
Not sure how it got introduced, I changed it back to 1.10 in latest patch though.